CONTRIBUTING.md

```diff
@@ -87,69 +87,158 @@ You may need to install the dev_requirements for this
 pip install -r path/to/source/dev_requirements
 ```
 
-_Hub:_  
-`pytest azext_iot/tests/iothub/test_iot_ext_unit.py`
+Example unit tests runs:
 
-_DPS:_  
-`pytest azext_iot/tests/dps/test_iot_dps_unit.py`
+_Hub:_
+`pytest azext_iot/tests/iothub/core/test_iot_ext_unit.py`
+
+_DPS:_
+`pytest azext_iot/tests/dps/core/test_dps_discovery_unit.py`
+
+Unit tests end in "_unit" so execute the following command to run all unit tests,
+`pytest -k "_unit"`
+
+Execute the following command to run the IoT Hub unit tests:
+
+`pytest azext_iot/tests/iothub/ -k "_unit"`
 
 ### Integration Tests
 
 Integration tests are run against Azure resources and depend on environment variables.
 
 #### Azure Resource Setup
 
-1. Create IoT Hub
-
-    > IMPORTANT: Your IoT Hub must be created specifically for integration tests and must not contain any devices when the tests are run.
+The following resources will be needed for the integration tests.
 
-2. Create Files Storage - In IoT Hub, click Files, create a new Storage Account and link to an empty Container.
+- IoT Hub
+- IoT Device Provisioning Service
+- IoT Digital Twin
```

Comment:
```suggestion
- Azure Digital Twins instance
```
The official marketing for the Azure Digital Twins service usually doesn't have an IoT prefix https://docs.microsoft.com/en-us/azure/digital-twins/overview